test/System.CommandLine.StaticCompletions.Tests/System.CommandLine.StaticCompletions.Tests.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net10.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <IsTestProject>true</IsTestProject>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- None of these deps should be hard-coded, but something's wonky unless I opt into the whole massive Microsoft.NET.TestFramework mess.
+         I don't want that. This should be able to stand alone. -->
```

Comment:
I still don't fully understand the TestFramework avoidance situation. But as it stands, everything running through Helix in this repo should use it. And if it doesn't work well, we should fix it.